src/data/pages.js

```diff
@@ -1,23 +1,71 @@
 import { gql } from '@apollo/client';
 
+export const PAGE_FIELDS = gql`
+  fragment PageFields on Page {
+    children {
+      edges {
+        node {
+          id
+          slug
+          uri
+          ... on Page {
+            id
+            title
+          }
+        }
+      }
+    }
+    id
+    menuOrder
+    parent {
+      node {
+        id
+        slug
+        uri
+        ... on Page {
+          title
+        }
+      }
+    }
+    slug
+    title
+    uri
+  }
+`;
+
+export const QUERY_ALL_PAGES_INDEX = gql`
+  ${PAGE_FIELDS}
+  query AllPagesIndex {
+    pages(first: 10000, where: { hasPassword: false }) {
+      edges {
+        node {
+          ...PageFields
+        }
+      }
+    }
+  }
+`;
+
+export const QUERY_ALL_PAGES_ARCHIVE = gql`
+  ${PAGE_FIELDS}
+  query AllPagesIndex {
+    pages(first: 10000, where: { hasPassword: false }) {
+      edges {
+        node {
+          ...PageFields
+        }
+      }
+    }
+  }
+`;
+
```

Comment:
Just wondering these two could become one 🤔

Reply:
so, yeha, totally, my thinking here was that since im kind of establishing a pattern of the index/archive/all that it made sense to include both, that way someone doesn't try to use one and it's not there and get confused? idk, thats just my thinking, also it makes it easy for someone to add new fields to one of them

im gonna roll with it as is but lmk if you think that doesnt make sense, im not opposed to just killing it

Reply:
Makes sense, for the clarity of the pattern ✌️